modules/yieldlabBidAdapter.js

```diff
@@ -67,16 +67,16 @@ export const spec = {
           width: sizes[0],
           height: sizes[1],
           creativeId: '' + matchedBid.id,
-          dealId: matchedBid.did,
+          dealId: matchedBid.pid,
```

Comment:
May want to add a test that verifies `pid` is being mapped to `dealId` as you expect.   Would also help verify that this was an intended change!

Reply:
Thank you for your comment. I agree and added a commit: https://github.com/prebid/Prebid.js/pull/2231/commits/4010d702062ecaaf8320348caa383b2ab425e517

Reply:
looks good, thx.  will get this merged.